src/components/molecules/EarthEditor/LayerTreeViewItem/LayerActions/index.tsx

```diff
@@ -66,10 +68,21 @@ const LayerActions: React.FC<Props> = ({
         </HelpButton>
       </Action>
       <Action disabled={!rootLayerId} onClick={importLayer}>
-        <HelpButton descriptionTitle={t("Add Layer.")} balloonDirection="bottom">
+        <HelpButton descriptionTitle={t("Add Layer")} balloonDirection="bottom">
```

Comment:
Please update all other HelpButton's text to have no period and have each word uppercased